### PR TITLE
MS-2573 Add html to `UserExamQuestion`

### PR DESCRIPTION
**Pull request by jasonewall**

### CODE REVIEW

**Comment by darkhelmet** [comment]
:ship:

**Comment by darkhelmet** [comment]
Well, you know, green first.

File: .travis.yml [hunk 1]
```diff
@@ -14,4 +14,8 @@ script: time ./script/travis.sh
 branches:
     only: master
 
+sudo: required
+dist: precise
+group: legacy
+
```
Comment (jasonewall) [inline .travis.yml]: @darkhelmet Le sigh

Comment (darkhelmet) [inline .travis.yml]: lol

**Pull request merged**